pkgs/development/python-modules/crc/default.nix

```
@@ -0,0 +1,17 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "crc";
+  version = "4.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-W1ElEk1127TSXsEHvaU/32zapdY8dH82CWdBNY9r2j4=";
+  };
+
+  meta = with lib; {
+    description = "Python module for calculating and verifying predefined & custom CRC's";
+    homepage = "https://nicoretti.github.io/crc/";
+    license = licenses.bsd2;
```

```suggestion
    license = licenses.bsd2;
    maintainers = with maintainers; [ ];
```

We require maintainer for new python packages.

---

Merged.